pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/service/adminoperation/LegacyKitResampleServiceTest.java

```diff
@@ -0,0 +1,222 @@
+    private static final String legacyShortId = "LEGACY_SHORT";
+    private static String esIndex;
+    private static DDPInstanceDto ddpInstanceDto;
+    private static DDPInstanceDao ddpInstanceDao = new DDPInstanceDao();
+    private static String ddpParticipantId = "PT_SAMPLE_QUEUE_TEST";
+    private static ParticipantDto participantDto = null;
+    private static int participantCounter = 0;
+    private static Pair<ParticipantDto, String> legacyParticipantPair;
+    public static TestKitUtil testKitUtil;
+    private static final String newCollaboratorSampleId = "NEW_SAMPLE_ID";
+    private static final String newCollaboratorParticipantId = "NEW_PARTICIPANT_ID";
+
+    private static final String oldCollaboratorSampleId = "OLD_SAMPLE_ID";
+    private static final String oldCollaboratorParticipantId = "OLD_PARTICIPANT_ID";
+    private static final String ddpKitRequestId = "RESAMPLE_KIT_REQUEST_ID";
+    private static List<ParticipantDto> participants = new ArrayList<>();
+    private static List<String> createdKits = new ArrayList<>();
+
+    private static LegacyKitResampleService legacyKitResampleService = new LegacyKitResampleService();
+    private static DDPInstance ddpInstance;
+
+    @BeforeClass
+    public static void doFirst() {
+        testKitUtil = new TestKitUtil(instanceName, instanceName, "resample", instanceName, "SALIVA", null);
+        testKitUtil.setupInstanceAndSettings();
+        esIndex = ElasticTestUtil.createIndex(instanceName, "elastic/lmsMappings.json", null);
+        ddpInstanceDto = ddpInstanceDao.getDDPInstanceByInstanceName(instanceName).orElseThrow();
+        ddpInstanceDto.setEsParticipantIndex(esIndex);
+        ddpInstanceDao.updateEsParticipantIndex(ddpInstanceDto.getDdpInstanceId(), esIndex);
+        ddpParticipantId = TestParticipantUtil.genDDPParticipantId(ddpParticipantId);
+        legacyParticipantPair = TestParticipantUtil.createLegacyParticipant(ddpParticipantId, participantCounter++, ddpInstanceDto,
+                shortId, ddpParticipantId, legacyShortId);
+        participantDto = legacyParticipantPair.getLeft();
+        ElasticTestUtil.createParticipant(esIndex, participantDto);
+        participants.add(participantDto);
+        ddpInstance = DDPInstance.getDDPInstance(instanceName);
+        String dsmKitRequestId = testKitUtil.createKitRequestShipping(ddpParticipantId, oldCollaboratorSampleId,
+                oldCollaboratorParticipantId, null,  ddpKitRequestId, "SALIVA", ddpInstance, "100");
+        createdKits.add(dsmKitRequestId);
+    }
+
+    @AfterClass
+    public static void tearDown() {
+        participants.forEach(participantDto ->
+                TestParticipantUtil.deleteParticipant(participantDto.getRequiredParticipantId()));
+        createdKits.forEach(kitRequestId -> testKitUtil.deleteKitRequestShipping(Integer.parseInt(kitRequestId)));
+        testKitUtil.deleteGeneratedData();
+        ElasticTestUtil.deleteIndex(esIndex);
```

Comment:
You are not getting rid of the DDP instance.
See my comments above about how to properly create it.